x-pack/plugins/observability_solution/apm/server/routes/services/annotations/get_stored_annotations.ts

```diff
@@ -70,7 +70,7 @@ export function getStoredAnnotations({
           type: AnnotationType.VERSION,
           id: hit._id as string,
           '@timestamp': new Date(hit._source['@timestamp']).getTime(),
-          text: hit._source.message,
+          text: hit._source.message! ?? hit._source.annotation.title,
```

Comment:
do we need this `!`?

```suggestion
          text: hit._source.message ?? hit._source.annotation.title,
```